scripts/py_matter_idl/matter_idl/data_model_xml/handlers/__init__.py

```diff
@@ -12,11 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+
 from matter_idl.matter_idl_types import Idl
 
 from .base import BaseHandler
 from .context import Context
 from .handlers import ClusterHandler
+from .parsing import NormalizeName
+
+LOGGER = logging.getLogger('data-model-xml-data-parsing')
+
+
+def contains_valid_cluster_id(attrs) -> bool:
```

Comment:
What's the type of `attrs`. The IDs should be checked after auto-conversion (base 0 int conversion that manages 0x and decimal)

Reply:
Added more typing (originally I had attrs untyped everywhere, but now I started typing them).

For now I would keep just length as I saw no evidence that the XML parse would place 0 or 0x0 here, so I am explicitly using string length only.

If we start having different data sets, I will update the logic.